app/src/main/res/layout/item_comment_audio.xml

```diff
@@ -0,0 +1,125 @@
+                app:layout_constraintTop_toTopOf="parent"
+                bind:imageError="@{@drawable/ic_person_black_24dp}"
+                bind:imageUrl="@{commentViewModel.createUser.photoUrl}"
+                />
+
+            <TextView
+                android:id="@+id/textView"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="@dimen/dp_8"
+                android:layout_marginStart="@dimen/dp_8"
+                android:layout_marginTop="@dimen/dp_8"
+                android:lines="1"
+                android:text="@{commentViewModel.createUser.userName}"
+                android:textColor="@android:color/black"
+                android:textSize="@dimen/sp_16"
+                android:textStyle="bold"
+                app:layout_constraintLeft_toRightOf="@+id/circleImageView"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                />
+            <TextView
+                android:id="@+id/textView2"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="@dimen/dp_8"
+                android:layout_marginStart="@dimen/dp_8"
+                android:lines="1"
+                android:textColor="@color/color_grey_700"
+                android:textSize="@dimen/sp_12"
+                app:layout_constraintLeft_toRightOf="@+id/circleImageView"
+                app:layout_constraintTop_toBottomOf="@+id/textView"
+                bind:milisecond="@{commentViewModel.createdAt}"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintHorizontal_chainStyle="spread_inside"
+                />
+
+
+            <android.support.constraint.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/dp_56"
+                android:layout_margin="@dimen/dp_8"
+                android:background="@drawable/bg_record_item"
+                app:layout_constraintTop_toBottomOf="@+id/circleImageView"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                >
+
+                <ImageView
+                    android:id="@+id/image_playing_audio"
+                    android:layout_width="@dimen/dp_24"
+                    android:layout_height="@dimen/dp_24"
+                    app:srcCompat="@drawable/ic_pause_circle_outline_black"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    android:layout_marginLeft="@dimen/dp_8"
+                    android:layout_marginEnd="@dimen/dp_8"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 15) you should *also* add `android:layout_marginRight="@dimen/dp_8"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.